veascan-subgraph-outbox/src/VeaOutbox.ts

```
@@ -68,6 +69,27 @@ export function handleChallenged(event: Challenged): void {
 }
 
+export function handleVerificationStarted(event: VerificationStarted): void {
+  const ref = getRef(event.address);
+  for (
+    let i = ref.totalClaims.minus(BigInt.fromI32(1));
+    i.ge(BigInt.fromI32(0));
+    i = i.minus(BigInt.fromI32(1))
+  ) {
+    const claimId = event.address.toHexString() + "-" + i.toString();
+    const claim = Claim.load(claimId);
+    if (claim && claim.epoch.equals(event.params._epoch)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)